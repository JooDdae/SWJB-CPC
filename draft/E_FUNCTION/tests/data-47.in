43
10 43 26 16 20 34 28 42 33 8 41 37 24 11 9 40 32 18 29 12 38 7 27 2 30 19 22 6 13 14 5 35 21 23 17 31 36 3 4 39 25 -1
11 23 17 13 40 4 3 31 27 25 12 7 36 32 24 6 43 10 15 37 29 14 41 8 42 21 9 38 26 30 2 22 33 -1
10 4 11 21 17 8 40 -1
36 23 18 22 3 21 26 12 13 20 9 10 30 38 6 4 39 31 42 11 5 41 15 34 -1
21 9 3 -1
6 39 -1
5 11 27 13 23 18 40 6 29 41 16 17 21 42 12 4 22 39 35 30 34 15 33 9 25 3 24 8 43 20 32 38 26 37 19 2 31 10 14 7 36 -1
35 41 30 18 34 25 42 40 37 36 33 23 27 24 16 19 28 7 32 2 17 13 4 43 6 31 21 9 10 15 22 -1
7 13 35 28 5 6 16 8 38 42 37 22 29 31 15 41 11 23 17 9 36 26 -1
26 6 4 33 35 42 10 31 18 8 20 3 7 2 39 38 30 9 36 41 11 40 43 -1
11 34 29 13 2 3 26 22 14 28 37 27 16 21 18 35 40 41 38 17 8 7 10 6 12 4 30 39 -1
14 5 6 29 12 28 8 10 31 25 15 -1
24 2 42 10 43 11 40 19 5 35 20 18 28 6 29 34 8 23 38 17 37 41 3 30 25 15 22 12 21 27 36 14 9 13 26 4 33 16 39 31 7 32 -1
7 28 10 39 15 26 5 43 18 19 17 22 35 25 13 37 30 29 24 9 36 3 33 16 42 40 21 6 27 8 23 41 12 38 31 2 32 4 14 -1
7 20 33 15 2 8 27 14 22 39 34 5 21 43 4 23 30 18 37 19 9 42 29 25 16 24 -1
9 22 11 19 29 24 2 40 32 16 38 -1
12 17 7 16 24 21 6 41 19 43 38 31 33 28 20 4 36 2 15 -1
12 8 38 39 31 23 42 21 41 24 17 35 2 20 15 11 25 9 10 30 22 16 32 37 6 36 18 33 3 29 4 34 7 26 43 28 19 13 -1
32 13 12 30 -1
28 6 27 29 34 35 11 -1
25 31 10 33 32 9 24 8 41 26 19 14 2 28 3 42 40 30 36 12 43 20 5 17 4 18 39 -1
41 15 38 7 10 22 20 3 13 25 21 17 40 37 29 6 28 11 34 9 4 16 32 42 12 31 39 30 -1
15 39 16 5 43 20 34 25 4 22 17 40 42 -1
25 33 23 15 37 32 36 27 6 20 38 26 22 7 29 14 24 12 9 43 8 3 41 17 40 13 35 42 30 -1
41 15 43 22 42 -1
3 34 17 27 36 20 24 4 13 16 33 22 2 26 37 32 21 38 5 39 35 11 -1
18 36 7 28 12 25 9 37 35 43 29 3 11 15 17 30 39 19 8 5 34 31 38 13 2 42 6 26 10 -1
6 17 8 36 38 4 42 24 29 7 22 3 43 9 11 13 10 18 30 40 31 41 21 12 2 5 16 20 -1
26 39 16 8 11 7 41 -1
5 39 22 4 27 6 28 38 24 33 42 8 19 16 20 36 9 7 13 25 32 21 2 18 31 30 23 11 37 43 10 12 15 41 17 40 29 14 -1
31 14 34 37 11 43 16 24 33 22 20 19 2 40 -1
33 19 -1
36 14 32 29 37 34 16 10 18 17 -1
9 21 20 24 2 14 -1
38 13 26 4 33 19 8 35 16 3 34 2 -1
3 39 36 32 8 -1
32 2 -1
8 25 41 7 35 23 39 30 31 26 19 21 10 28 13 27 33 22 16 20 6 29 -1
7 4 9 43 15 18 16 30 32 28 38 13 41 23 42 14 33 11 17 24 25 21 37 40 -1
34 25 11 43 9 39 4 30 19 3 5 2 15 13 18 20 21 17 10 35 12 36 8 14 22 23 6 41 24 26 -1
9 13 43 11 17 4 3 7 22 24 14 30 2 25 36 39 21 23 10 42 8 15 31 32 18 5 26 20 27 -1
22 8 21 18 20 41 28 11 6 19 30 43 27 15 29 33 38 -1
22 4 30 12 6 42 36 11 39 33 31 40 14 32 2 41 5 21 8 13 3 16 15 20 23 -1
