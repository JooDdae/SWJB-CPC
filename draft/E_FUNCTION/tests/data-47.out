YES
3 10 26 